.github/CONTRIBUTING.md

```diff
@@ -212,7 +212,29 @@ invoke `make format`.
 The target `make formatall` in the root will automatically run all style checks
 and make any required changes. PRs must pass the formatting checks before landing.
 
-### Step 8: Landing
+### Step 8: Licensing
+
+Tock is dual-licensed under Apache2/MIT. Files in Tock require a short header
+indicating this. In 99% of cases, the following text should be at the top of every file:
+
+> © Copyright 2020[-20XX] The Tock Project Authors.
+>
+> Dual licensed under the Apache License, Version 2.0 or the MIT License, at your option.
+> See the License for the specific language governing permissions and limitations under the License.
+> This file may not be copied, modified, or distributed except according to those terms.
+
+If you create a new "version or edition" of a given work, then the date range
+should be extended to include the newest work. In practice, this means any
+update to a source file will require updating the range (and some churn
+hopefully largely restricted to January each year).
+
+For simplicity, we ask that copyright should always be held by the Tock Project
+Authors. We are happy to recognize new contributors in the
```

Comment:
"For simplicity, we ask that copyright should always be held by the Tock Project Authors."

This wording implies copyright assignment, which is not what we want here.

Reply:
I just removed that sentence. I guess my question is will/should the CI look at who is listed as the author, or just ignore that field?